modules/mbedtls/register_types.cpp

```diff
@@ -30,6 +30,7 @@
 
 #include "register_types.h"
 
+#include "core/config/project_settings.h"
 #include "crypto_mbedtls.h"
 #include "dtls_server_mbedtls.h"
 #include "packet_peer_mbed_dtls.h"
```

Comment:
We normalize group includes like this in modules: module files first, then core Godot files, then thirdparty. (Suggestion not complete as I couldn't include line 37 in it, it's outside the changed diff.)
```suggestion
#include "crypto_mbedtls.h"
#include "dtls_server_mbedtls.h"
#include "packet_peer_mbed_dtls.h"

#include "core/config/project_settings.h"
```